CMakeLists.txt

```diff
@@ -625,13 +625,11 @@ set(includedir \${prefix}/${CMAKE_INSTALL_INCLUDEDIR})
 set(PACKAGE_NAME ${PROJECT_NAME})
 set(PACKAGE_VERSION ${arpack_ng_VERSION})
 set(PACKAGE_URL "https://github.com/opencollab/arpack-ng/")
-# TODO: Fill these in with something appropriate.
-#set(LAPACK_LIBS)
-#set(BLAS_LIBS)
+set(LAPACK_LIBS ${LAPACK_LIBRARIES})
+set(BLAS_LIBS ${BLAS_LIBRARIES})
```

Comment:
This results in:
```
Libs.private: /usr/lib/liblapack.so;/usr/lib/libblas.so /usr/lib/libblas.so
```

and
```
$ PKG_CONFIG_PATH=$PWD pkg-config  --libs --static arpack
-larpack /usr/lib/liblapack.so\;/usr/lib/libblas.so /usr/lib/libblas.so 
```

Either way it is wrong in both cmake and autotools, since you should instead be using:
```
Requires.private: lapack blas
```
For documentation on the pkg-config format, see https://jlk.fjfi.cvut.cz/arch/manpages/man/pc.5#PROPERTY_KEYWORDS or https://www.mankier.com/5/pc#Description-Property_Keywords

Reply:
Two problems with Requires.private:
* Assumes that whatever lapack/blas library you're linking against will have a corresponding .pc file.  The netlib reference implementation didn't have these until recently, and I'm guessing a lot of other implementations don't have them yet.
* Suppose our chosen lapack/blas implementation does have .pc files, we would need to get the name of its .pc files and use them in place of "lapack blas".